edx_proctoring/templates/proctoring/seq_proctored_exam_entrance.html

```diff
@@ -0,0 +1,75 @@
+{% load i18n %}
+<div class="sequence proctored-exam entrance" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Would you like to take {{ display_name }} as a Proctored Exam?
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      Since you're enrolled in this course as a verified student, you have the option to take this exam with
```

Comment:
"As a verified learner in this course, you have the option of taking this exam with online proctoring. To be eligible to earn credit for this course, you must take and pass the proctoring review for this exam."
@chrisndodge @pbaruah Same question as above re "proctoring review" term -- is it OK or are there preferred alternatives?